### PR TITLE
Support for HT-A7000

### PR DESCRIPTION
**Pull request by Flameeyes**
This builds on top of the other pull request I sent to fix typing, to include some additional detail in structures, and provide better access to the content change structure.

### CODE REVIEW

**Comment by rytilahti** [comment]
Could you please rebase this on top of the current master? Looks otherwise good to me, but I cannot test this personally right now.

File: songpal/notification.py [hunk 1]
```diff
@@ -153,23 +159,11 @@ def __str__(self):
...
     """This gets sent as a notification when the source changes."""
 
     make = classmethod(make)
 
-    contentKind = attr.ib()
-    service = attr.ib()
-    source = attr.ib()
-    output = attr.ib()
-    uri = attr.ib()
-    applicationName = attr.ib()
```
Comment (rytilahti) [inline songpal/notification.py:166]: Not all of these seem to be available in `PlayInfo`, at least `applicationName` seems to be missing.

Comment (Flameeyes) [reply songpal/notification.py:166]: Turns out `applicationName` was the only one missing and it was meant to exist in `PlayInfo` as well.

Added as a pre-requisite commit.

**Review by rytilahti — APPROVED** [review]
LGTM, thanks! :+1:

**Pull request merged**